post_installation_configuration/multi-architecture-configuration.adoc

```diff
@@ -10,6 +10,12 @@ An {product-title} cluster with multi-architecture compute machines is a cluster
 
 For information on migrating your single-architecture cluster to a cluster that supports multi-architecture compute machines, see xref:../updating/migrating-to-multi-payload.adoc#migrating-to-multi-payload[Migrating to a cluster with multi-architecture compute machines].
 
+include::modules/multi-architecture-verifying-cluster-compatibility.adoc[leveloffset=+1]
+
+[role="_additional-resources"] 
+.Additional resources
+* xref:../updating/migrating-to-multi-payload.adoc#migrating-to-multi-payload[Migrating to a cluster with multi-architecture compute machines].
```

Comment:
This xref is missing an anchor ID https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#internal-cross-references